src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs

```diff
@@ -724,6 +731,21 @@ private static HubMessage BindInvocationMessage(string? invocationId, string tar
         return new InvocationMessage(invocationId, target, arguments, streamIds);
     }
 
+    private object? BindType(ref Utf8JsonReader reader, ReadOnlySequence<byte> input, Type type)
+    {
+        if (type == typeof(RawResult))
+        {
+            var start = reader.BytesConsumed;
+            reader.Skip();
+            var end = reader.BytesConsumed;
+            var sequence = input.Slice(start, end - start);
+            // Review: Technically we could pass the sequence without copying into a new array
```

Comment:
super nit
```suggestion
            // REVIEW: Technically we could pass the sequence without copying into a new array
```

Reply:
We should update this comment now to reflect we're not copying here.